### ISSUE TITLE
Bug: wrong canonical URL

### ISSUE THREAD

**Issue by lkrimphove** [Hashnode/starter-kit#100]
It seems like the wrong canonical URL gets added to the blog post.
I set a canonical URL in the Hashnode draft editor and published the article.
However, when I open the blog article and check the canonical URL I find the current URL of the article, instead of the given canonical URL.

**Comment by panda-sandeep** [comment]
Hi there! please raise a support ticket [here](https://hashnode.com/#support) to get quick response.

**Comment by lkrimphove** [comment]
I'm pretty sure that this is a problem with the starter kit and not Hashnode itself:
- If I open up the Hashnode blog the canonical URL is set correctly.
- If I open up the website I deployed using the starter kit the canonical URL is set to the current article URL. I tried this using the Personal and the Hashnode theme. The canonical URL wasn't correctly set on both of them.

When opening up `pages/[slug].tsx` I saw, that the canonical URL is set to `post.url`. I think this could be problem, but I'm not sure.

I'm sorry if this is actually not a problem with the starter kit but with Hashnode. Thank you four your time!
